gcloud-java-storage/src/main/java/com/google/gcloud/storage/spi/DefaultStorageRpc.java

```diff
@@ -505,7 +505,6 @@ public void write(String uploadId, byte[] toWrite, int toWriteOffset, long destO
       } else {
         range.append('*');
       }
-      httpRequest.getHeaders().setContentRange(range.toString());
```

Comment:
What's the rationale for deleting this line?  Looks like it's in the [master branch](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/gcloud-java-storage/src/main/java/com/google/gcloud/storage/spi/DefaultStorageRpc.java#L505).